lib/ex_plasma/utxo.ex

```diff
@@ -29,7 +29,7 @@ defmodule ExPlasma.Utxo do
   """
```

Comment:
a general question that came up while I was using ex_plasma...
when you're building a transaction you're using Utxo struct for inputs and ouputs but inputs have different mandatory fields than outputs do... are we invaliding these and erroring?

Reply:
Added checks for this in  6c32c79